pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -112,8 +111,9 @@ type zedagentContext struct {
 	subCertObjConfig          pubsub.Subscription
 	deferredChan              <-chan time.Time
 	FlowlogQueue              chan<- *flowlog.FlowMessage
-	TriggerDeviceInfo         chan<- struct{}
-	TriggerHwInfo             chan<- struct{}
+	TriggerDeviceInfo         chan<- infoDest
+	TriggerHwInfo             chan<- infoDest
+	TriggerCloudLocationInfo  chan<- infoDest
```

Comment:
All these Trigger* fields can start with lower-case, zedagent context is not exported (I realize this mistake was there already).

Reply:
Yep.